src/serialization/schema_validator.hpp

```diff
@@ -112,20 +111,21 @@ class schema_validator : public abstract_validator
 		}
 	};
 
+	/** Controls the way to print errors. */
+	bool create_exceptions_;
```

Comment:
Needs a Wreorder warning to be fixed in the .cpp file

Reply:
Fixed in 38c099d.